src/scss/_table.scss

```diff
@@ -0,0 +1,320 @@
+  }
+
+  .table-base td {
+    display: block;
+
+    font-size: 0.8em;
+    text-align: right;
+
+    border-bottom: 1px solid var(--color-bg-light);
+  }
+
+  .table-base td::before {
+    float: left;
+
+    font-weight: bold;
+
+    content: attr(data-label);
+  }
+
+  .table-base td:last-child {
+    border-bottom: 0;
+  }
+}
+
+// end table-base
+
+// style below this should be removed when integrating new table with the rest of the application
+.table {
+  width: 100%;
+  margin: 1rem 0;
+  padding-bottom: 0.5rem;
+
+  font-size: 1rem;
+
+  background: var(--color-bg-dark);
+}
+
+.table .loading {
+  display: flex;
+
+  margin-top: 0.5em;
+  margin-left: 0.4em;
+}
+
+.table .row.preview {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.table .headers {
+  position: relative;
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+  margin-top: 0.3em;
+  padding: 0.3em 1rem;
+  padding-left: 1rem;
+
+  border-bottom: 1px solid var(--color-bg-light);
+}
+
+.table .header {
+  flex: 1;
+
+  font-size: 1.2rem;
+}
+
+.hooks.table .headers {
+  padding-left: 4.25rem;
+}
+
+.hook-status {
+  fill: none;
+}
+
+.hook-status.-success {
+  stroke: var(--color-green);
+}
+
+.hook-status.-failure {
+  stroke: var(--color-red);
+}
+
+.table .row .cell {
+  flex: 1;
+  align-items: center;
+  justify-content: center;
+  width: 300px;
+
+  font-weight: lighter;
+  text-align: center;
+  word-wrap: break-word;
+}
+
+.table .headers .header.-last {
+  display: flex;
+  justify-content: flex-end;
+  padding-right: 1rem;
+}
+
+.table .cell.-last {
+  display: flex;
+  justify-content: flex-end;
+  padding-right: 1rem;
+
+  button {
+    margin: 0.25rem 0;
+  }
+}
+
+.table .source-id {
+  min-width: 330px;
+}
+
+.table .headers .first-cell {
+  width: 2rem;
+}
+
+.filler-cell {
+  width: 6ch;
+}
+
+.table .headers .-label {
+  color: var(--color-primary);
+  font-size: 1.5rem;
+}
+
+.table .cell.source-id {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.table .text {
+  flex: 1;
+  padding: 3px 12px;
+
+  text-align: center;
+
+  background: var(--color-bg);
+}
+
+.table .details {
+  margin: 1rem;
+}
+
+.preview .status.success {
+  color: var(--color-green);
+}
+
+.preview .status.failure {
+  color: var(--color-red-light);
+}
+
+.table .info {
+  display: flex;
+  display: inline-block;
+  flex-direction: row;
+
+  width: 100%;
+  padding: 0.4em 1em;
+
+  font-size: 14px;
+
+  border-bottom: 1px solid var(--color-bg-light);
+  border-left: 2px solid var(--color-bg-light);
+}
+
+.table .info:last-child {
+  border-bottom: none;
+}
+
+.table .info.-pending {
+  border-left-color: var(--color-bg-light);
+}
+
+.table .info.-running {
+  border-left-color: var(--color-yellow);
+}
+
+.table .info.-success {
+  border-left-color: var(--color-green);
+}
+
+.table .info.-failure,
+.table .info.-error {
+  border-left-color: var(--color-red);
+}
+
+.table .element span.-m-r,
```

Comment:
i think we're using `.-m-r` beyond the scope of this PR, so we'll address changing it to utility class in a separate PR probably.

should that `span` in this case have it's own class?

Reply:
see comment above, anything below/attached to `.table` will be gutted and removed with hooks refactor